### DISCUSSION TITLE
1.0.0-beta.15

### DISCUSSION THREAD

**Discussion by mrkai77** [mrkai77/Loop#294] [Announcements]
## 📰 Important News
- 💸 Reminder to support me & Loop by [sponsoring the project](https://github.com/sponsors/MrKai77)!
- 💄 Can't donate? The next best way to support the project is to make some **app icons** for Loop!
- 🌐 **Localization:** If you are willing to translate the app into other languages and haven't already notified me, please open a new issue on GitHub :)

## ✨ New Features
- ✨ #263 More options in custom window actions. Use exact coordinates, preserve frames and more! (@MrKai77)
- ✨ #258 Use Loop when caps lock is enabled (@MrKai77)
- ✨ #233 Ability to exclude certain apps from Loop, such as virtual machines (@just-doit)
- ✨ #232 Add haptic feedback to window snapping (@MrKai77)
- ✨ #207 Don't truncate numbers in number input (@MrKai77)
- ✨ #201 Option to not make the window Looped the active window (@MrKai77)
- ✨ #188 Ability to disable cursor interaction. Only for those of you who exclusively use keybinds :) (@MrKai77)
- ✨ #183 Ability to hide radial menu (@MrKai77)
- ✨ #175 Ability to disable haptic feedback (@cyborg-joshi)
- ✨ #165 Remove preset cycle keybinds, in favour of new cycling keybind system! (@MrKai77)
- ✨ #155 Adjust Size Feature: enlarge or shrink a window's frame directly from Loop! (@MrKai77)
- ✨ #146 Support `AXEnhancedUserInterface` Accessibility Attribute. Apps such as Notion & Firefox will now resize smoother! (@MrKai77)
- ✨ #94 Custom window padding on each side of screen (@MrKai77)

## 🐛 Bug Fixes
- 🐛 #261 Make Loop work with Hyperkey.app (@MrKai77)
- 🐛 #238 Disable window snapping on widgets (@MrKai77)
- 🐛 #229 Remove Sparkle sandboxing configuration (@MrKai77)
- 🐛 #219 Can‘t restore window frame on drag (@MrKai77)
- 🐛 #208 Some custom cycles are broken (@MrKai77)
- 🐛 #197 Settings window opens itself randomly (@MrKai77)
- 🐛 #196 Skip "Next/Previous Screen" if no screen detected (@MrKai77)
- 🐛 #187 "Double-click trigger key to trigger Loop" recognized incorrectly (@MrKai77)
- 🐛 #180 False snap recognition on multiple screens (@MrKai77)

## 🥳 Notable Mentions
- 💄 #293 Refactor UI (@MrKai77)
- ⚡️ #291 Stage Manager optimization (@phlpsong)
- ⚡️ #290 Use enum in Settings window (@phlpsong)
- 💫 #282 Make radial menu animations smoother (@MrKai77)
- ✏️ #280 Fixed typo in "receive" (@adamnemecek)

**Full Changelog**: https://github.com/MrKai77/Loop/compare/1.0.0-beta.14...1.0.0-beta.15

**Comment by fxzer** [comment]
体验了一下，觉得动画挺丝滑，但是我是 Rectangle 老用户，尝试着用 Loop 替换 Rectangle，因为   Rectangle 将窗口显示在上、下一屏幕是(Cmd+option+Control + ↔️，发现快捷键只能录制一个，做不到无成本迁移，但是发现希望改进一下

**Reply by mrkai77** [reply]
Please comment in English or else I won't understand :)

**Reply by fxzer** [reply]
👌，Let me translate it：After experiencing it, I found the animation to be quite smooth, but as an old user of Rectangle, I tried to replace it with Loop because Rectangle displays the window on the top and next screens using (Command+option+Control)+ ↔️， I found that only one shortcut key can be recorded, making it impossible to achieve cost free migration. However, I have found that I hope to improve it

**Reply by mrkai77** [reply]
You can set up screens functionality in they keybindings tab: 
<img width="500" alt="image" src="https://github.com/MrKai77/Loop/assets/68963405/ea3cbea9-a8d2-4e93-be71-618f722093c7">


When combined in a cycling keybind, you can achieve a pretty similar functionality to Rectangle :)
<img width="500" alt="image" src="https://github.com/MrKai77/Loop/assets/68963405/38456a50-d0e3-4bf8-9202-ba67782cdae8">

Maybe I'm not fully understanding, but you are correct, a cost free migration is currently impossible in Loop.
